articles/appliance/index.html

```diff
@@ -0,0 +1,137 @@
+---
+url: /appliance
+section: appliance
+classes: topic-page
+---
+
+<div class="topic-page-header">
+  <div data-name="example" class="topic-page-badge"></div>
+  <h1>Auth0 Appliance</h1>
+  <p>
+    An Auth0 deployment that exists in a dedicated area of Auth0's cloud, your cloud, or your own data center.
+  </p>
+</div>
+
+<h2>What is Auth0 Appliance?</h2>
+<p>
+  Auth0 Appliance is a managed service that is typically used if your organization's compliance or policy requirements prevent you from using a multi-tenant cloud service.
+</p>
+<p>
+  Auth0 is responsible for installation, maintenance, patching and updates. You are responsible for supplying and monitoring the infrastructure on which Auth0 runs (including, but not limited to, the VM host, storage, network resources, and other required dependencies).
+</p>
+
+<h2>Appliance Planning</h2>
+
+<ul class="topic-links">
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/appliance-overview">Overview of the Auth0 Appliance</a>
+    <p>
+      The Auth0 Appliance is an option for your organization when compliance or other policy requirements prevent you from using a multi-tenant cloud service.
+    </p>
+    <ul>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/appliance/geo-ha">Overview of the Geographic High-Availability Appliance</a>
+      </li>
+    </ul>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/modules">Appliance Deployment Modules</a>
+    <p>
+      For security reasons, rules and custom database connections for Auth0 appliances run in a JavaScript sandbox. You can use the full power of the ECMAScript 5 language and a few selected libraries.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/infrastructure">Appliance Infrastructure Requirements</a>
+    <p>
+      This document contains information about the Appliance and its infrastructure requirements.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/appliance-sprint">Appliance Sprint (Customer Onboarding)</a>
```

Comment:
@nkohari The URL does not work for me. I can see that the doc is available at 
https://auth0-docs-content-pr-2316.herokuapp.com/docs/onboarding/appliance-sprint so we should either update the link to `/onboarding/appliance-sprint` or add a redirect. Not sure which is preferable.